Another projekts/calculator/main_calculator_1.py

```diff
@@ -0,0 +1,37 @@
+run = True
+
+while run:
+    what_todo = input("What do you want to do?\n1 for +\n2 for -\n3 for *\n4 for /\n: ")
+
+    def mathematical_operation():
+        first_number = int(input("Enter first number: "))
+        second_number = int(input("Enter second number: "))
+        if what_todo == '1':
+            result = first_number + second_number
+        elif what_todo == '2':
+            result = first_number - second_number
+        elif what_todo == '3':
+            result = first_number * second_number
+        elif what_todo == '4':
+            result = first_number / second_number
+        print(f"Answer is {result}")
```

Comment:
Isn't it better to output "**The** answer is ..."?

Reply:
> Isn't it better to output "**The** answer is ..."?

Have you done it yet?

Reply:
> > Isn't it better to output "**The** answer is ..."?
> 
> Have you done it yet?

Yes, i did it

Reply:
Thanks for the feedback.